pybikes/nextbike.py

```diff
@@ -26,29 +28,38 @@ class Nextbike(BikeShareSystem):
         'company': 'Nextbike GmbH'
     }
 
-    def __init__(self, tag, meta, domain, city_uid, hostname = 'nextbike.net'):
+    def __init__(self, tag, meta, domain, city_uid, hostname = 'nextbike.net', bbox = None):
         super(Nextbike, self).__init__(tag, meta)
         self.url = BASE_URL.format(hostname=hostname, domain=domain)
         self.uid = city_uid
+        self.bbox = None
+        if bbox is not None:
+            self.bbox = box(bbox[0][0], bbox[0][1], bbox[1][0], bbox[1][1])
```

Comment:
on most cases, unless you want a strict checking `if bbox is not None` == `if bbox`